frontend/src/components/mine/ContactInfo/PartyRelationships/DefaultContact.js

```diff
@@ -79,7 +81,24 @@ export const DefaultContact = (props) => (
         <br />,
         <br />,
         <h6>{props.partyRelationshipTitle} Since</h6>,
-        <span>{formatDate(props.partyRelationship.start_date) || "Unknown"}</span>,
+        <span>
+          {formatDate(props.partyRelationship.start_date) || "Unknown"}
+          {props.partyRelationship.mine_party_appt_type_code === "MMG" &&
+            props.partyRelationship.documents.length > 0 && (
+              <span>
+                {" "}
```

Comment:
these white spaces can be deleted and just have `-`

Reply:
I tried that. With the `<LinkButton>` tag right there, it doesn't seem to work. Prettier adds those 🙂